.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,3 +38,16 @@ The changelogs will be integrated by the core maintainers after the merge.  See
 <!-- Comment:
 If you need an accelerated review process by the community (e.g., for critical bugs), mention @jenkinsci/code-reviewers
 -->
+
+### Maintainer checklist
+
+Before the changes is marked as `ready-for-merge`: 
+
+- [ ] There is at least 2 approvals for the pull request and no outstanding requests for change
```

Comment:
This is not exactly aligned with what the `CONTRIBUTING` file is saying:

> Usually we merge pull requests after 2 votes from reviewers or after 1 vote and 1-week delay without extra reviews.

Reply:
Yes, needs alignment. Though I would rather require 2+ votes, because now we have more active reviewers that it was before